Update orders-api-overview.md (#868)
bigcommerce/dev-docs: docs/api-docs/orders/orders-api-overview.md

<div class="HubBlock-content">

> ### Note

> > * The example above contains the minimum required fields for a [create order request](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/createanorder).

Reviewer: ```suggestion
> * The example above contains the minimum required fields for a [create order] (https://developer.bigcommerce.com/api-reference/store-management/orders/orders/createanorder) request.
```

> > * The example above contains the minimum required fields for a [create order] (https://developer.bigcommerce.com/api-reference/store-management/orders/orders/createanorder) request.

Reviewer 2: ```suggestion
> * The example above contains the minimum required fields for a [create order](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/createanorder) request.
```

> ### Note
> * If not specified, `status_id` defaults to `1`.
> * The refunded status is neither paid nor unpaid.

> > * See [Order Statuses](https://support.bigcommerce.com/s/article/Order-Statuses#rename) in the Help Center for information on changing `custom_label` in the control panel.

Reviewer: ```suggestion
> * For information on changing `custom_label` in the control panel, see [Order Statuses](https://support.bigcommerce.com/s/article/Order-Statuses#rename).
```

> ### Note
> * Custom products do not get added to the catalog.

> > * If a price is not specified, and you use the store's product catalog price; override this price with the `price_inc_tax` and `price_ex_tax`.

Reviewer: I think the proposed change altered the meaning of that sentence. Perhaps, we can rewrite it to make it more comprehensible?
```suggestion
> * If you do not specify the product price in the [create order](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/createanorder request, the product's catalog price will be used. You can override this price with the `price_inc_tax` and `price_ex_tax`.
```

> ### Note
> * Abbreviated state names (ex: `CA` instead of `California`) in an order address will cause tax document submission to fail.

> > * You can calculate taxes using rules specified in the store (unless [automatic taxes](https://support.bigcommerce.com/s/article/Automatic-Tax-Setup) are enabled).

Reviewer: ```suggestion
> * You can calculate taxes using rules specified in the store unless [automatic taxes](https://support.bigcommerce.com/s/article/Automatic-Tax-Setup) are enabled.
```

<div class="HubBlock-content">

> ### Note

> > * Depending on the payment method, different information will be available (not all payment gateways return the full card or fraud detail).

Reviewer: ```suggestion
> * Not all payment gateways return the full card or fraud detail. Depending on the payment method, different information will be available.
```

## Calculating totals

> Order `subtotal` and `total` calculate automatically; edits to the following properties trigger a recalculation.

Reviewer: The table starting on line 602 does not render correctly. I think it should be removed as the same properties are covered in the table above (line 590).

```suggestion
Order `subtotal` and `total` calculate automatically; edits to the following properties trigger a recalculation.
```

|`billing_address`|
|`shipping_addresses`|

> You can override calculated values such as product prices, subtotals, and totals by sending a fixed value in the request. If you do not supply values for these properties, you will automatically calculate them based on the preset store values and tax rules.

Reviewer: I think that passive voice is fine in this instance since users do not calculate values themselves.

**What is the difference between country_ISO2 and country?**

There is no requirement to specify country when you specify `country_ISO2` in the shipping and billing addresses and vice versa.

**How can I take payment for an order?**

You can either process payment through a third party or using the control panel.

**Can I generate a shipping quote from a carrier using the API?**

> Not at this time. If you create an order either in the control panel or via API, it returns a 204 when trying to get a Shipping Quote.

Reviewer: ```suggestion
Not at this time. If you create an order either in the control panel or via API, it returns a 204 when trying to get a shipping quote.
```
